.golangci.yml

```diff
@@ -0,0 +1,29 @@
+linters:
+  enable-all: false
+  disable-all: true
+  enable:
+    - deadcode
+    - errcheck
+    - gocyclo
+    - gofmt
+    - goimports
+    - golint
+    - gosimple
+    - govet
+    - ineffassign
+    - interfacer
+    - lll
+    - misspell
+    - nakedret
+    - staticcheck
+    - structcheck
+    - typecheck
+    - unconvert
+    - unparam
+    - unused
+    - varcheck
+linters-settings:
+  gocyclo:
+    min-complexity: 16
```

Comment:
Was going to ask why `min` but found an associated issue: https://github.com/golangci/golangci-lint/issues/368